README.md

```diff
@@ -106,7 +106,7 @@ language = "[C|C++]"
 style = "[Both|Type|Tag]"
 # How the generated documentation should be commented.
 # C uses /* */; C++ uses //; Doxy is like C but with leading * per line.
```

Comment:
Can you update this comment too?

Reply:
Yeah, although it looks like C++ comment was wrong in the first place? (it uses `///` not `//`)

Reply:
Updated both.